scalyr_agent/builtin_monitors/syslog_monitor.py

```diff
@@ -1380,7 +1380,7 @@ def _initialize(self):
         self.__server_list = self.__build_server_list(self._config.get("protocols"))
 
         # our disk logger and handler
-        self.__disk_logger = None
+        self._disk_logger = None
```

Comment:
Add a comment that we are making this protected for testing purposes.

Reply:
Returned back the mangled version.